consensus/core/src/leader_scoring_strategy.rs

```diff
@@ -31,17 +26,11 @@ pub(crate) trait ScoringStrategy: Send + Sync {
 pub(crate) struct CertifiedVoteScoringStrategyV2 {}
 
 impl ScoringStrategy for CertifiedVoteScoringStrategyV2 {
-    fn calculate_scores_for_leader(
-        &self,
-        subdag: &UnscoredSubdag,
-        leader_slot: Slot,
-        committer: &BaseCommitter,
-    ) -> Vec<u64> {
+    fn calculate_scores_for_leader(&self, subdag: &UnscoredSubdag, leader_slot: Slot) -> Vec<u64> {
         let num_authorities = subdag.context.committee.size();
         let mut scores_per_authority = vec![0_u64; num_authorities];
 
-        let wave = committer.wave_number(leader_slot.round);
-        let decision_round = committer.decision_round(wave);
+        let decision_round = leader_slot.round + DEFAULT_WAVE_LENGTH - 1;
```

Comment:
I'm leaving a TODO to merge the DEFAULT_WAVE_LENGTH and MINIMUM_WAVE_LENGTH into a single `WAVE_LENGTH` const. We are unlikely to change them via config or even statically, because a lot of logic does not actually support WAVE_LENGTH > 3.